tests/Jellyfin.Naming.Tests/ExternalFiles/ExternalPathParserTests.cs

```diff
@@ -0,0 +1,98 @@
+using System.Text.RegularExpressions;
+using Emby.Naming.Common;
+using Emby.Naming.ExternalFiles;
+using MediaBrowser.Model.Dlna;
+using MediaBrowser.Model.Globalization;
+using Moq;
+using Xunit;
+
+namespace Jellyfin.Naming.Tests.ExternalFiles;
+
+public class ExternalPathParserTests
+{
+    private readonly ExternalPathParser _audioPathParser;
+    private readonly ExternalPathParser _subtitlePathParser;
+
+    public ExternalPathParserTests()
+    {
+        var englishCultureDto = new CultureDto("English", "English", "en", new[] { "eng" });
+        var frenchCultureDto = new CultureDto("French", "French", "fr", new[] { "fre", "fra" });
+
+        var localizationManager = new Mock<ILocalizationManager>(MockBehavior.Loose);
+        localizationManager.Setup(lm => lm.FindLanguageInfo(It.IsRegex(@"en.*", RegexOptions.IgnoreCase)))
+            .Returns(englishCultureDto);
+        localizationManager.Setup(lm => lm.FindLanguageInfo(It.IsRegex(@"fr.*", RegexOptions.IgnoreCase)))
+            .Returns(frenchCultureDto);
+
+        _audioPathParser = new ExternalPathParser(new NamingOptions(), localizationManager.Object, DlnaProfileType.Audio);
+        _subtitlePathParser = new ExternalPathParser(new NamingOptions(), localizationManager.Object, DlnaProfileType.Subtitle);
+    }
+
+    [Theory]
+    [InlineData("", false)]
+    [InlineData("MyVideo.srt", false)]
+    [InlineData("MyVideo.mka", true)]
+    public void ParseFile_AudioFile_ReturnsPathWhenAudio(string path, bool valid)
```

Comment:
I'd split this into 2 tests, one with the valid data and the other with invalid data